server/global/images_importation/imagesAnayzer.js

```diff
@@ -0,0 +1,129 @@
+import { queryPromise } from "../../db/database.js";
+import { removeExtension } from "../Files.js";
+import { addErrorTitle } from "../Logger.js";
+
+import {
+  normalizeDCI,
+  getInvalidNormalizedDci,
+} from "../molecules_importation/moleculesAnalyzer.js";
+
+const VALID_FORMATS = ["jpeg", "jpg", "svg", "png"];
+export const isFormatValid = (filename) =>
+  new RegExp(`\\.${VALID_FORMATS.join("|")}\\s*$`, "i").test(filename);
```

Comment:
You could store the regex in a constant, as it we won't regeneate it each time the function is called

Reply:
done! 👍